config/routes.rb

```diff
@@ -25,6 +25,7 @@
       resources :moves, only: %i[index show create destroy update] do
         resources :documents, only: %i[create destroy]
         resources :events, only: %i[create], controller: 'move_events'
+        resources :journeys
```

Comment:
As we did for move, better to be explicit here and specify all the possible actions.

Reply:
good point 👍

Reply:
Does the journeys controller implement all of the restful actions? If so I think it's fine to keep this as is.

Reply:
it does not implement `delete`, so no, its not a completely restful

Reply:
In which case the list should be explicit I think

Reply:
now fixed 👍